styles/index.css

```diff
@@ -30,20 +37,89 @@
     border:0;
     margin:0 3px;
     padding:3px;
-    border-radius: 4px;
 }
 
 .active-nav-pill{
-    background-color:var(--secondary-clr);
+    border-bottom:solid 2px var(--secondary-clr);
 }
 
 .nav-pill:hover{
-    background-color:var(--secondary-clr);
     transform: translateY(-2px);
     translate: transform 3s ease;
+    border-bottom:solid 2px var(--secondary-clr);
 }
 
 .nav-pill-img{
     margin:2px 8px;
     width:1.6rem;
+}
+
+.banner-img{
+    border-radius:1rem;
+    width:50rem;
+}
+
+.why-us-contanier{
+    width:40rem;
+    padding:1rem;
+}
+
+.glass-bg, .category-container{
+    padding:3px;
+    margin:5px;
+    width:3rem;
+    box-shadow: rgba(0, 0, 0, 0.15) 0px 2px 8px;
+    border-radius: 10px;
+    border: 1px solid rgba( 255, 255, 255, 0.18 );
+    transition: all .5s ease;
+}
+
+.category-container{
+    width:8rem;
+}
+
+.glass-bg:hover, .category-container:hover{
+    box-shadow: rgba(0, 0, 0, 0.15) 0px 2px 8px;
+    transform: translate3D(0,-1px,0) scale(1.03);
+}
+
```

Comment:
Same in these both classes same properties for everything for hover too Then why different classes I don't get your point of doing so

Reply:
Since the glass-gb has a width declared, I wasn't able to over write the style by placing an another class which could get the category-container to 8rem width. That's why had to make this kind of declaration.